packages/lab/src/progress/CircularProgress/CircularProgress.tsx

```diff
@@ -30,10 +24,6 @@ export interface CircularProgressProps extends HTMLAttributes<HTMLDivElement> {
    * Default value is 100.
    */
   max?: number;
-  /**
-   * If `true`, the info panel will be displayed.
-   */
-  showInfo?: boolean;
```

Comment:
Props removal needs to be mentioned in changesset. If below is meant for this, needs better wording to associate the two
> Remove indeterminate (show no progress value) Progress

Reply:
updated